src/P3.jl

```diff
@@ -21,12 +24,12 @@ import Thermodynamics as TD
 import Thermodynamics.Parameters as TDP
 
 const PSP3 = CMP.ParametersP3
+using CloudMicrophysics.Parameters:
+    MassPowerLaw, AreaPowerLaw, SlopeLaw, SlopePowerLaw, SlopeConstant
```

Comment:
Same here. Would prefer to keep the qualifier to see in the code where the parameters are coming from